applications/directory/src/components/ProfileDetails.js

```diff
@@ -14,7 +15,7 @@ const displayUrlReg = /http(s)?(:)?(\/\/)?|(\/\/)?(www\.)?/;
 
 const socialIcons = {
   website: <LanguageIcon />,
-  behance: <SvgIcon component={BehanceIcon} />,
+  behance: <SvgIcon component={BehanceIcon} />, // TODO - it does not show the icon
```

Comment:
To fix SVG display, can you try solutions 1 and 2 here, in order? https://stackoverflow.com/a/56418369/83897

Here is an example of modifying webpack loader rules in Next.js: https://github.com/neatowebsolutions/greatupsells/blob/master/applications/shopify-admin/next.config.js#L48